### PR TITLE
docs: Fix some code snippets for unification strategies

### PR DESCRIPTION
**Pull request by tomaarsen**
# Description

Resolve the immediately-fixable issues from #3807, but does not immediately close that issue, as the `unified_responses` don't exist anymore, but are still listed in the docs. cc @davidberenstein1957 

**Type of change**

- [x] Documentation update

**How Has This Been Tested**

It hasn't.

**Checklist**

- [ ] I added relevant documentation
- [x] I followed the style guidelines of this project
- [x] I did a self-review of my code
- [x] I made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the `CHANGELOG.md` file (See https://keepachangelog.com/)

---

- Tom Aarsen

### CODE REVIEW

**Comment by alvarobartt** [comment]
Indeed, did `unified_responses` exist at some point? I think it has always been a private attribute of `FeedbackRecord`, but maybe I missed something? 🤔

**Comment by tomaarsen** [comment]
I think it was a public parameter in a PR that david made to introduce them, but also made private later in that same PR.

**Review by alvarobartt — APPROVED** [review]

File: docs/_source/_common/tabs/unfication_strategies.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
 :::{tab-item} LabelQuestion
 
 ```python
-from argilla import RatingQuestionStrategy, FeedbackRecord
+from argilla import RatingQuestionStrategy, FeedbackDataset
```
Comment (alvarobartt) [inline docs/_source/_common/tabs/unfication_strategies.md]:
```suggestion
from argilla import LabelQuestionStrategy, FeedbackDataset
```

File: docs/_source/_common/tabs/unfication_strategies.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ dataset.records[0].unified_responses
 :::{tab-item} MultiLabelQuestion
 
 ```python
-from argilla import RatingQuestionStrategy, FeedbackRecord
+from argilla import RatingQuestionStrategy, FeedbackDataset
```
Comment (alvarobartt) [inline docs/_source/_common/tabs/unfication_strategies.md]:
```suggestion
from argilla import MultiLabelQuestionStrategy, FeedbackDataset
```

**Pull request merged**